python_modules/libraries/dagster-airflow/dagster_airflow/dagster_asset_factory.py

```diff
@@ -94,6 +95,11 @@ def find_upstream_dependency(node_name: str) -> None:
     return (output_mappings, keys_by_output_name, internal_asset_deps)
 
 
+@superseded(
+    additional_warn_text=(
+        "The `dagster-airlift` library is no longer best practice. Use the `dagster-airlift` library instead."
```

Comment:
comment is wrong! `dagster-airflow` for the first one.

Maybe "`load_assets_from_airflow_dag` has been superseded by the functionality in the `dagster-airlift` library"?

Reply:
Updated in [d51d30c](https://github.com/dagster-io/dagster/pull/26863/commits/d51d30cf2e9867cab04ac56d1c35ba8fd8152fda)